src/renderer/shaders/symbolizer/symbolizerFragmentShader.glsl

```diff
@@ -19,6 +20,7 @@ void main(void) {
     }else{
         c = symbolColor;
     }
+    c.a *= filtering;
```

Comment:
ok, so now the filter is applied or not when dealing with colors, right?

Reply:
The filtering affects the opacity everytime (instead of just when the `color:` was overriding the symbol color

Reply:
thx for your explanation!